ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -219,6 +219,7 @@ Item {
                 d.swapFormData.selectedAccountIndex = d.selectedAccountIndex
                 d.swapFormData.selectedNetworkChainId = StatusQUtils.ModelUtils.getByKey(RootStore.filteredFlatModel, "layer", 1, "chainId")
                 d.swapFormData.fromTokensKey = tokensKey
+                d.swapFormData.toTokenKey = RootStore.areTestNetworksEnabled ? Constants.swap.testStatusTokenKey : Constants.swap.mainnetStatusTokenKey
```

Comment:
`RootStore.areTestNetworksEnabled` should be coming from the respective adaptor's `SwapStore`, no? (same below)

Reply:
in this case, its the WalletLayouts flow and not really Swap yet and hence we use the same store use in the WalletLayout.